src/main/scala/com/twitter/diffy/DiffyServiceModule.scala

```diff
@@ -73,6 +75,12 @@ object DiffyServiceModule extends TwitterModule {
   var httpsPort =
     flag[String]("httpsPort", "443", "Port to be used when using HTTPS as a protocol")
 
+  var maxRequestSize =
```

Comment:
These requirements are independent of SSL and are generally useful. Please use these settings in a shared trait between Http and Https. Bonus points for adding this support at a lower finagle layer so that it may be directly configured in DifferenceProxy to support Thirft in addition to http and https.